Fixes a build failure with SIP 4.19.6 (possibly other versions > 4.19.2):
    sip: ::KFontChooser ctor argument 5 has an unsupported type for a Python signature - provide a valid type, %MethodCode and a C++ signature

Obtained from: https://anonscm.debian.org/cgit/pkg-kde/kde-sc/pykde4.git/commit/debian/patches?id=9de587a159f47d30068c84127c06917594eaf2e0
--- sip/kdeui/kfontchooser.sip
+++ sip/kdeui/kfontchooser.sip
@@ -59,7 +59,7 @@ public:
 
     typedef QFlags<KFontChooser::DisplayFlag> DisplayFlags;
 
-    explicit                KFontChooser (QWidget* parent /TransferThis/ = 0, const KFontChooser::DisplayFlags& flags = KFontChooser::DisplayFrame, const QStringList& fontList = QStringList(), int visibleListSize = 8, Qt::CheckState* sizeIsRelativeState = 0) [(QWidget* = 0, const KFontChooser::DisplayFlags& = KFontChooser::DisplayFrame, const QStringList& = QStringList(), int = 8, Qt::CheckState* = 0)];
+    explicit                KFontChooser (QWidget* parent /TransferThis/ = 0, const KFontChooser::DisplayFlags& flags = KFontChooser::DisplayFrame, const QStringList& fontList = QStringList(), int visibleListSize = 8, Qt::CheckState sizeIsRelativeState = Qt::Unchecked) [(QWidget* = 0, const KFontChooser::DisplayFlags& = KFontChooser::DisplayFrame, const QStringList& = QStringList(), int = 8, Qt::CheckState* = 0)];
 %MethodCode
 Py_BEGIN_ALLOW_THREADS
     sipCpp = new sipKFontChooser (a0, *a1, *a2, a3, &a4);
--- sip/kdeui/kfontdialog.sip
+++ sip/kdeui/kfontdialog.sip
@@ -28,7 +28,7 @@ class KFontDialog : KDialog
 
 
 public:
-    explicit                KFontDialog (QWidget* parent /TransferThis/ = 0, const KFontChooser::DisplayFlags& flags = KFontChooser::NoDisplayFlags, const QStringList& fontlist = QStringList(), Qt::CheckState* sizeIsRelativeState = 0) [(QWidget* = 0, const KFontChooser::DisplayFlags& = KFontChooser::NoDisplayFlags, const QStringList& = QStringList(), Qt::CheckState* = 0)];
+    explicit                KFontDialog (QWidget* parent /TransferThis/ = 0, const KFontChooser::DisplayFlags& flags = KFontChooser::NoDisplayFlags, const QStringList& fontlist = QStringList(), Qt::CheckState sizeIsRelativeState = Qt::Unchecked) [(QWidget* = 0, const KFontChooser::DisplayFlags& = KFontChooser::NoDisplayFlags, const QStringList& = QStringList(), Qt::CheckState* = 0)];
 %MethodCode
 Py_BEGIN_ALLOW_THREADS
     sipCpp= new sipKFontDialog (a0, *a1, *a2, &a3);
